packages/installations/src/helpers/idb-manager.ts

```diff
@@ -25,18 +25,27 @@ const DATABASE_NAME = 'firebase-installations-database';
 const DATABASE_VERSION = 1;
 const OBJECT_STORE_NAME = 'firebase-installations-store';
 
-let dbPromise: Promise<DBWrapper> | null = null;
-function getDbPromise(): Promise<DBWrapper> {
+interface InstallationsDB extends DBSchema {
+  'firebase-installations-store': {
+    key: string;
+    value: InstallationEntry | undefined;
```

Comment:
Do we prefer this style of optionals instead of `value?: InstallationEntry;`?

Reply:
It would be better to do `value?: InstallationEntry` but unfortunately we're extending a type from the external `idb` library and that type doesn't allow `value` to be an optional property (but does allow `| undefined`):

https://github.com/jakearchibald/idb/blob/231942c8a6b425e799e22885afb88751df4432ca/src/entry.ts#L124